mayaUSD_Win.mod.template

```diff
@@ -0,0 +1,42 @@
++ USD ${USD_VERSION} ${USD_INSTALL_LOCATION}
+icons: 
+plug-ins: 
+presets: 
+scripts: 
+resources: 
+PYTHONPATH+:=lib/python
+PATH+:=bin
+PATH+:=lib
```

Comment:
Windows specific mod template which just has these extra PATH lines. Otherwise it's identical.